src/main/java/com/fulcrumgenomics/util/PickLongIndices.scala

```diff
@@ -64,31 +64,62 @@ private class IndexSet extends Iterable[Index] {
   def contains(index: String): Boolean = this.set.contains(index)
 }
 
+/** A Metric class for information about each picked index. */
+case class IndexMetric(index: String,
+                       source: String,
+                       min_mismatches:Int,
+                       indices_at_min_mismatches: Int,
+                       gc: Double,
+                       min_delta_g: Option[Double]) extends Metric
```

Comment:
I am not sure if reading and writing a metric with an option parameter works.  It's probably a good idea to add a test to the metric class?